packages/manager/core/generator/app/templates/src/App.tsx

```diff
@@ -1,18 +1,11 @@
 import React, { useEffect, useContext } from 'react';
-import { QueryClientProvider, QueryClient } from '@tanstack/react-query';
+import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import { odsSetup } from '@ovhcloud/ods-common-core';
 import { ShellContext } from '@ovh-ux/manager-react-shell-client';
 import { RouterProvider, createHashRouter } from 'react-router-dom';
-import { Routes } from './routes';
-
-const queryClient = new QueryClient({
-  defaultOptions: {
-    queries: {
-      staleTime: 300_000,
-    },
-  },
-});
+import { Routes } from './routes/routes';
+import queryClient from './queryClient';
```

Comment:
I think it's not a good idea : people may think it is ok to import queryClient directly instead of getting it from the context with the hook `useQueryClient`.

Reply:
👍